src/public/target/sections/CancerBiomarkers/Section.js

```diff
@@ -1,9 +1,93 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
-import FilterTable from './custom/FilterTable';
+import _ from 'lodash';
 
-const Section = ({ ensgId, symbol, data }) => (
-  <FilterTable symbol={symbol} ensgId={ensgId} />
-);
+import { Link } from 'ot-ui';
+
+import Table from '../../../common/Table/Table';
+import { PaginationActionsComplete } from '../../../common/Table/TablePaginationActions';
+import { label } from '../../../../utils/global';
+import { naLabel } from '../../../../constants';
+
+import { loader } from 'graphql.macro';
+import useBatchDownloader from '../../../../hooks/useBatchDownloader';
+const BIOMARKERS_QUERY = loader('./sectionQuery.gql');
+
+const columns = [
+  {
+    id: 'id',
+    label: 'Biomarker',
+  },
+  {
+    id: 'diseases',
+    label: 'Disease',
+    renderCell: row =>
+      row.disease ? (
+        <Link to={`/disease/${row.disease.id}`}>{label(row.disease.name)}</Link>
+      ) : (
+        <>{naLabel}</>
+      ),
+    exportValue: row => (row.disease ? label(row.disease.name) : naLabel),
+  },
+  {
+    id: 'drugName',
+    label: 'Drug',
+  },
+
+  {
+    id: 'associationType',
+    label: 'Association',
+    renderCell: row => _.capitalize(row.associationType.replace(/_/g, ' ')),
+  },
+  {
+    id: 'evidenceLevel',
+    label: 'Evidence',
+  },
+  {
+    id: 'sources',
+    label: 'Sources',
+    renderCell: row => {
+      return (
+        <>
+          {row.sources.map((source, i) => (
+            <Link key={i} external to={source.link}>
+              {source.name}
+            </Link>
+          ))}
+        </>
+      );
+    },
+    exportValue: row => row.sources.map(source => source.name).join(),
+  },
+];
+
+const Section = ({ ensgId, symbol, data }) => {
+  const [rows, setRows] = useState([]);
+
+  const getAllBiomarkers = useBatchDownloader(
+    BIOMARKERS_QUERY,
+    {
+      ensgId: ensgId,
+    },
+    'data.target.cancerBiomarkers'
+  );
+
+  useEffect(() => {
+    getAllBiomarkers().then(d => {
+      setRows(d);
+    });
+  }, []);
+
+  return (
+    <Table
```

Comment:
The cancer biomarkers API provides server side paging: ` cancerBiomarkers(page: Pagination): CancerBiomarkers`
We could take advantage of this server side paging by trying to only the get the necessary rows needed for a particular page. Here's a section that takes advantage of this server side paging: https://github.com/opentargets/platform-app/blob/alpha/src/public/target/sections/RelatedTargets/Section.js

Reply:
Thanks @mirandaio I agree that server side pagination in this case is a good idea.
However, to my understanding that would not work with table search/filtering: 
the API endpoint `cancerBiomarkers(page: Pagination)` takes *only* the pagination parameter, unlike the `knownDrugs` that also takes care of table filtering:
```knownDrugs(
freeTextQuery: String
size: Int
cursor: [String!]
): KnownDrugs```

I don't mind either way - @andrewhercules what do you think: search or load/pagination performance?

Reply:
The [Cancer Biomarkers dataset](https://www.cancergenomeinterpreter.org/biomarkers) is quite small - it does not need to be server-side pagination. I believe the target with the most rows is [ERBB2](https://alpha.targetvalidation.org/target/ENSG00000141736) with 104. 

To keep most data tables consistent, I would ignore the server-side pagination and allow for search. But if there's a performance or code issue, then I defer to your expertise and we can implement the server-side pagination instead.

Reply:
@LucaFumis you are right, you cannot use searching/filtering on a server side mode table if there is no support in the backend, as you don't have all the rows on the client. And it looks like CancerBiomarkers does not provide anything besides pagination.

As Andrew said, it looks like there is not a lot of rows in those queries, so you can keep the client side table implementation.

Reply:
If backend is not planning to provide a `freeTextQuery` like in known drugs, then yeah, in that case, client side paging in the only option.